package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "lint": "standard",
-    "unit": "cross-env TS_NODE_PROJECT=./templates/app-ts/tsconfig.json tap test/*.test.js templates/app/test/*/*.test.js templates/app-ts/test/*/*.test.ts",
+    "unit": "cross-env TS_NODE_PROJECT=./templates/app-ts/tsconfig.json FASTIFY_BOOT_PATH=./templates/app/app.js tap -R classic -j1 --no-esm test/generate-typescript.test.js templates/app/plugins/*/test/*.test.js templates/app/routes/*/test/*.test.js templates/app-ts/test/*/*.test.ts",
```

Comment:
why is `FASTIFY_BOOT_PATH` needed?
what is `-R classic`?
Please copy the `.taprc` from https://github.com/fastify/fastify/blob/master/.taprc if you want to make tests faster.

Also, why `-j 1`? Parallel testing should make things a bit faster.

Reply:
`FASTIFY_BOOT_PATH` is not longer relevant, I'll remove

`-R classic` makes newer tap use the same reporter we're used to, it's an automatic reflex for me to use it

`-j1` also re-enables streaming (so the tests progress updates as tests complete). Parallel in tap now forces batching, which creates laggy output and degrades devex imo. 

Will do re .taprc but I propose we add -R classic, and possibly -j1 as well.

Reply:
caught up on phone, -R classic is going into taprc, -j1 isn't